src/site/_includes/layouts/_base.njk

```diff
@@ -57,7 +57,7 @@
       'analytics_storage': 'denied'
     });
     gtag('js', new Date());
-    gtag('config', window.googleTagId);
+    gtag('config', window.googleTagId, { debug_mode: window.location.host !== 'design-system.service.mod.gov.uk'  });
```

Comment:
Are we able to use a var for this?

(We do have `baseUrl` but that includes `https://`. Could do something like `(new URL('{{ vars.baseUrl }}')).host` but will need the polyfill loaded.)

Reply:
I don't have access to the polyfill here because `main.js` hasn't run at this point

Reply:
Could we do it in `src/site/_data/vars.js` though if we move `baseUrl` to a variable?